### PR TITLE
Small fix of libtinyiiod

### PR DESCRIPTION
**Pull request by cristiantpop**

### CODE REVIEW

File: parser.c [hunk 1]
```diff
@@ -88,7 +88,6 @@ static int parse_open_string(struct tinyiiod *iiod, char *str)
 {
 	char *device, *ptr;
 	long samples_count;
-	unsigned int i;
```
Comment (mirceacaprioru) [inline parser.c:90]: There are some misspellings in the commit message: loock, iioosciloscope, allready

Comment (stefpopa) [reply parser.c:90]: `Signed-off-by` is missing

Comment (mirceacaprioru) [reply parser.c:90]: I think libiio is the one sending the commands iio-scope is just the application using it. This is just a guess i could be wrong.

**Comment by stefpopa** [comment]
I would add libtinyiiod prefix to the commit messages, for example: `libtinyiiod: Add new line before return`.
The rest looks good to me

**Comment by stefpopa** [comment]
Hmm... Initially this PR had 3 commits, now it has 6 :). Please add a changelog after force-push. Also, as a rule of thumb, the preferred maximum number of commits in a pull request is 5. However, more commits are fine, as long as the changes are plain and obvious.

**Comment by amiclaus** [comment]
PR description is empty, please add it, summarizing the commits.

File: tinyiiod.c [hunk 2]
```diff
@@ -157,8 +157,8 @@ void tinyiiod_do_write_attr(struct tinyiiod *iiod, const char *device,
 	char buf[128];
```
Comment (stefpopa) [inline tinyiiod.c]: I would rename this commit:
`libtinyiiod: Fix out of bounds array access`
Also, add a short description in the commit message.

File: ci/travis/build_project.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (stefpopa) [inline ci/travis/build_project.sh]: I am not sure, does this commit belong in this PR?

**Comment by stefpopa** [comment]
Maybe we can squash commit 9a0f21a and b3d4a27 into a single commit which deals with fixing style issues and in the commit message we can describe what was actually fixed: new line after return, blank spaces, etc

File: parser.c (hunk 1 above)
Comment (cristiantpop) [reply parser.c:90]: Fixed problems here

**Review by commodo — APPROVED** [review]

File: tinyiiod.c (hunk 2 above)
Comment (cristiantpop) [reply tinyiiod.c]: "libtinyiiod" prefix was added
PR had 3 commits, now it has 6 :)....: this was my mistake trying to solve a conflict, fixed now
PR description is empty, please add it, summarizing the commits.: I will add this
libtinyiiod: Fix out of bounds array access: this is another pull request

File: ci/travis/build_project.sh (hunk 3 above)
Comment (cristiantpop) [reply ci/travis/build_project.sh]: this was my mistake trying to solve a conflict, fixed now

**Comment by commodo** [comment]
[LGTM](https://i.redd.it/kof0ov0erih21.jpg)

**Review by stefpopa — APPROVED** [review]
LGTM

**Pull request merged**